common/util.go

```diff
@@ -518,3 +519,13 @@ func ParseAccelDevices(devices string, acceleration ffmpeg.Acceleration) ([]stri
 	}
 	return strings.Split(devices, ","), nil
 }
+
+func ParseEthAddr(strJsonKey string) (string, error) {
+	var keyJson map[string]interface{}
+	if err := json.Unmarshal([]byte(strJsonKey), &keyJson); err == nil {
+		if address, ok := keyJson["address"].(string); ok {
```

Comment:
```suggestion
		if address, ok := keyJson["address"]; ok {
```

Reply:
After changing to what is recommended, the JSON parsing failed at runtime. I reverted it back and it is working fine: https://github.com/eliteprox/go-livepeer/commit/81e876bc860c9a7fb6735f99caa6524235404040

Please recommend if there is a better way to handle this

Reply:
Could you write how it failed for you?

I tried changing the code to the following:
```
     var keyJson map[string]string
        if err := json.Unmarshal([]byte(strJsonKey), &keyJson); err == nil {
            if address, ok := keyJson["address"]; ok {
                    return address, nil
            }
    }
```

And it worked for me. Could you write the command (or test) that fails in your case?

Reply:
I tried this and the test passed, but my functional tests failed. I determined the difference was the test `TestParse_ParseEthKeystorePathValidFile` did not include anything more than a single level JSON string with single address field. I've updated the test to include the entire JSON structure from keystore. You will find that the suggested code will fail on test. 

Thanks for the help, any further suggestions to simplify it?